modules/auth/auth.controller.js

```diff
@@ -0,0 +1,296 @@
+// Package imports
+const nodemailer = require('nodemailer');
+const jwt = require('jsonwebtoken');
+const env = require('dotenv').config();
+const {default: to} = require('await-to-js');
+
+// Constant Imports
+const authConstants = require('./auth.constants');
+const globalConstants = require('../../constants/constants.js');
+
+// Model imports
+const User = require('./auth.model');
+
+// Logger import
+const logger = require('../../config/logger/logger');
+
+/**
+ * Created login controller to send response to the user on login
+ * @param {Object} req
+ * @param {Object} res
+ * @return {Object}
+ */
+const loginCtrl = async (req, res) => {
+  if (req.user) {
+    logger.info(authConstants.labels.loginCtrl.success, req.user);
+    return res.status(200).json({success: 1, message: globalConstants.responseMessages.logInUser.success, data: req.user});
+  } else {
+    logger.error(authConstants.labels.loginCtrl.failure, req);
+    return res.status(200).json({success: 0, message: globalConstants.responseMessages.logInUser.failure, data: null});
+  }
+};
+
+/**
+ * Created signup controller to send response to the user on signup
+ * @param {Object} req
+ * @param {Object} res
+ * @param {Object} next
+ * @return {Object}
+*/
+const signupCtrl = async (req, res, next) => {
+  if (req.user) {
+    logger.error(authConstants.labels.signupCtrl.success, req);
+    return res.status(200).json({success: 1, message: globalConstants.responseMessages.signUpUser.success, data: req.user});
+  } else {
+    logger.error(authConstants.labels.signupCtrl.failure, req);
+    return res.status(200).json({success: 0, message: globalConstants.responseMessages.signUpUser.failure, data: null});
+  }
```

Comment:
Please explain what are you doing in the controller function. `loginCtrl` and `signupCtrl` both controller functions are same with different names.

Reply:
The response is different in both controllers in terms of the message we are sending